examples/mapbox/package.json

```diff
@@ -9,8 +9,8 @@
     "global": "^4.4.0"
   },
   "devDependencies": {
-    "@cubejs-backend/bigquery-driver": "^0.23.8",
-    "@cubejs-backend/postgres-driver": "^0.23.8",
-    "@cubejs-backend/server": "^0.23.14"
+    "@cubejs-backend/bigquery-driver": "^0.24.5",
+    "@cubejs-backend/postgres-driver": "^0.24.5",
+    "@cubejs-backend/server": "^0.24.5"
```

Comment:
@YakovlevCoded Do we need to install cubestore-driver and remove the unnecessary driver here?